### ISSUE TITLE
Zoom Problem with version 2.5

### ISSUE THREAD

**Issue by john-davies** [mpetroff/pannellum#783]
I've just moved to using version 2.5 and I can't get the zoom feature to work. I was previously using some local copies of the version 2.4 css and js files and everything worked fine.

After having moved to version 2.5 I find that the zoom command ( mouse or onscreen ) don't respond at all and, checking on the Firefox developer console, no http requests are made when a zoom is invoked.

For version 2.5 I'm fetching the css and js remotely rather than storing locally, copying the lines from the multiresolution example:
```
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/pannellum@2.5.2/build/pannellum.css"/>
<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/pannellum@2.5.2/build/pannellum.js"></script>
```
I've tried using my local 2.4 copy of the js and css and everything works fine and http requests are made when I zoom.

The latest online example at https://pannellum.org/documentation/examples/multiresolution/ works fine for me but I'm using a cylindrical projection rather than equirectangular. I generated the multi-res files using the` -C -B -V 38` options to `generate.py` which has worked fine in the past.

The only other difference is that I'm now setting up and creating the viewer and panormama scenes from data downloaded from a JSON file rather than hard coding it. However from my results with the older 2.4 version I don't think that this is the problem.

**Comment by mpetroff** [comment]
Could you please post the contents of your JSON configuration file? It seems to acting like it's in the zoom limits.

**Comment by john-davies** [comment]
I think that I've managed to exclude the JSON configuration file from the issue.

This is the original version using local Panellum css/js files - 
https://thereteng.appspot.com/tair_carn_isaf_pano.html. The zoom works fine.

This is the same html file except that the local Panellum css/js files have been replaced with references to the online version - https://thereteng.appspot.com/tair_carn_isaf_pano_2_5.html. Now the zoom doesn't work.

The diff of the two versions is:
```
john www $ diff tair_carn_isaf_pano.html tair_carn_isaf_pano_2_5.html 
8,10c8,9
<     <link rel="stylesheet" href="css/pannellum.css"/>
<     <script type="text/javascript" src="js/pannellum.js"></script>
<     <script type="text/javascript" src="js/libpannellum.js"></script>
---
>     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/pannellum@2.5.2/build/pannellum.css"/>
>     <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/pannellum@2.5.2/build/pannellum.js"></script>


```

**Comment by mpetroff** [comment]
The behavior you're seeing is due to an interaction between your `minPitch` / `maxPitch` settings and the `avoidShowingBackground` setting, combined with a bug in the `multiResMinHfov` setting introduced in 35083170d535b8f2fdce4392052aa50c929880dc. With the `multiResMinHfov` setting at its default `false` value, the `minHfov` parameter is supposed to be ignored for multires panoramas. However, due to the bug, the `minHfov` parameter was actually used with `multiResMinHfov` set to `false` (and ignored when its set to `true`). Since the default `minHfov` parameter is 50, and the `avoidShowingBackground` `minHfov` was calculated to be ~49, this prevented any zooming.

I fixed this in 6fe51935c83d26d0e7ac4ccbb0ebf957835012c6 to match the documentation and intended behavior, so it will be fixed in 2.5.3, which will be released soon. The temporary fix is to set `multiResMinHfov` to `true`, but this will need to be removed once 2.5.3 is released.

**Comment by john-davies** [comment]
Thanks, I'll use the temporary fix with `multiResMinHfov` for now.

**Comment by mpetroff** [comment]
I just released 2.5.3 with the fix.

**Comment by john-davies** [comment]
Yes, that works fine. Thanks.
